internal/pkg/agent/transpiler/vars_test.go

```diff
@@ -80,6 +80,7 @@ func TestVars_Replace(t *testing.T) {
 			false,
 		},
 		{
+			// data will be resolved to other.data
```

Comment:
```suggestion
			// data will be resolved to other.data since 'other' is the default provider set at variable creation (see mustMakeVarsWithDefault call)
```

Reply:
Fixed.